crates/op-rbuilder/src/args.rs

```diff
@@ -0,0 +1,111 @@
+//! Additional Node command arguments.
+//!
+//! Copied from OptimismNode to allow easy extension.
+
+//! clap [Args](clap::Args) for optimism rollup configuration
+
+use std::{env::var, str::FromStr};
+
+use alloy_primitives::B256;
+use rbuilder::utils::Signer;
+use reth_optimism_node::args::RollupArgs;
+use serde::{Deserialize, Deserializer};
+use serde_with::DeserializeAs;
+
+/// Prefix for env variables in config
+const ENV_PREFIX: &str = "env:";
+
+/// Parameters for rollup configuration
+#[derive(Debug, Clone, Default, PartialEq, Eq, clap::Args)]
+#[command(next_help_heading = "Rollup")]
+pub struct OpRbuilderArgs {
+    /// Rollup configuration
+    #[command(flatten)]
+    pub rollup_args: RollupArgs,
+    /// Builder secret key for signing last transaction in block
+    #[arg(long = "rollup.builder-secret-key")]
+    pub builder_secret_key: Option<EnvOrValue<String>>,
```

Comment:
Since we are using flags already, we do not need to use the `env:` pattern from the rbuilder. It adds an extra layer of indirection that is not necessary. We can use env variables in clap [like this](https://rust.code-maven.com/slides/rust/clap-and-environment-variables.html) and simplify this code.

Reply:
why does the rbuilder use the env: pattern?